packages/protocol/contracts/layer1/tiers/TierProviderBase.sol

```diff
@@ -16,12 +16,27 @@ abstract contract TierProviderBase is ITierProvider {
     uint16 public constant GRACE_PERIOD = 240; // minutes
     uint96 public constant BOND_UNIT = 75 ether; // TAIKO tokens
 
+    /// @inheritdoc ITierProvider
+    function getTierIds() external pure returns (uint16[] memory ids_) {
```

Comment:
Still need some minor changes to adapt these not-yet-supported tiers. 
Let me do it.

Reply:
No worries, I can change it my way.

Reply:
I already submitted pr https://github.com/taikoxyz/taiko-mono/pull/18349/files to adapt this change.